etc/profile-a-l/armcord.profile

```diff
@@ -0,0 +1,43 @@
+# Firejail profile for armcord
+# Description: Standalone Discord client
+# This file is overwritten after every install/update
+# Persistent local customizations
+include armcord.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.config/ArmCord
+
+# sh is needed to allow Firefox to open links
+#include allow-bin-sh.inc
+
+# Modules might depend on nodejs.
+# Add the below lines to your armcord.local if you need this.
+# Allow node (disabled by disable-interpreters.inc)
+#include allow-nodejs.inc
+#private-bin node
+
+mkdir ${HOME}/.config/ArmCord
+whitelist ${HOME}/.config/ArmCord
+whitelist /opt/armcord
+whitelist /usr/share/armcord
+
+# The lines below are needed to find the default Firefox profile name, to allow
+# opening links in an existing instance of Firefox (note that it still fails if
+# there isn't a Firefox instance running with the default profile; see #5352)
+noblacklist ${HOME}/.mozilla
+whitelist ${HOME}/.mozilla/firefox/profiles.ini
+
+ignore novideo
+private-bin armcord
```

Comment:
```suggestion
include globals.local

# sh is needed to allow Firefox to open links
#include allow-bin-sh.inc

# Modules might depend on nodejs.
# Add the below lines to your armcord.local if you need this.
# Allow node (disabled by disable-interpreters.inc)
#include allow-nodejs.inc
#private-bin node

# The lines below are needed to find the default Firefox profile name, to allow
# opening links in an existing instance of Firefox (note that it still fails if
# there isn't a Firefox instance running with the default profile; see #5352)
noblacklist ${HOME}/.mozilla
whitelist ${HOME}/.mozilla/firefox/profiles.ini

noblacklist ${HOME}/.config/ArmCord

mkdir ${HOME}/.config/ArmCord
whitelist ${HOME}/.config/ArmCord
whitelist /opt/armcord
whitelist /usr/share/armcord

ignore novideo
private-bin armcord
```

To make it easier to isolate and read the normal profile entries, it might be
useful to put all the optional sections together.

Such as right after globals.local (and maybe also the `ignore` commands) or
right before the redirect.